### PR TITLE
TestCrossSiteScriptV2: fixed the wrongly placed log statement

### PR DESCRIPTION
**Pull request by KajanM**
log statement which will never get executed is moved to the proper place

### CODE REVIEW

File: src/org/zaproxy/zap/extension/ascanrules/TestCrossSiteScriptV2.java [hunk 1]
```diff
@@ -324,6 +324,8 @@ public void scan(HttpMessage msg, String param, String value) {
 								+ " onMouseOver=" + context.getSurroundingQuote() + "alert(1);", context,
 								HtmlContext.IGNORE_TAG);
 						if (contexts2 == null) {
+							log.debug("Failed to find vuln in with simple onmounseover "
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/ascanrules/TestCrossSiteScriptV2.java]: I'm not sure we should log that here, the `contexts2` is (usually) `null` if the scanner is stopped (we don't know if the attack failed or not, we didn't try/check).

File: src/org/zaproxy/zap/extension/ascanrules/TestCrossSiteScriptV2.java [hunk 2]
```diff
@@ -332,10 +334,6 @@ public void scan(HttpMessage msg, String param, String value) {
 									"", contexts2.get(0).getTarget(), contexts2.get(0).getMsg());
 							attackWorked = true;
 						}
-						if (!attackWorked) {
-							log.debug("Failed to find vuln in with simple onmounseover "
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/ascanrules/TestCrossSiteScriptV2.java]: This is executed (the unit tests cover this line)?

Comment (kingthorin) [reply src/org/zaproxy/zap/extension/ascanrules/TestCrossSiteScriptV2.java]: Confirmed:
image

Comment (KajanM) [reply src/org/zaproxy/zap/extension/ascanrules/TestCrossSiteScriptV2.java]: I am sorry, I didn't think of a case where context2 is not null and empty

Comment (kingthorin) [reply src/org/zaproxy/zap/extension/ascanrules/TestCrossSiteScriptV2.java]: Nonprobkem, thanks for looking into it.

**Pull request closed without merging**